### DISCUSSION TITLE
Pi OS - GUI install

### DISCUSSION THREAD

**Discussion by Bronkow** [Chia-Network/chia-blockchain#1130] [Q&A]
Hej there!

I just follwed the install Guide for the Pi OS (64 bit) and everything works fine - except the point that i can't start up the GUI. I just get the electron Window with the menu bar but no content

When I try to start the GUI it says
/home/pi/chia-blockchain/app.asar .unpacked/daemon
false
Running python script 
Script ../src/daemon/server.py
chid process success
…
No cert
Have cert
Daemon Server started

I use a pi4

Is the any chance of getting help by this bare infos? ^^
Cheers

**Comment by dorkmo** [comment]
Have you had any luck? I got it running on Ubunutu Pi Gui but on RC7 ive been having issues staying synced.

**Reply by Bronkow** [reply]
I had no success yet with the raspberry 64 bit. I still have a blank electron window. On an old 2010 MacBook Pro with ubuntu everything works fine with rc7.

**Comment by hoffmang9** [comment]
There were npm conflicts over the weekend that seem to be resolved upstream. Try re running `sh install-gui.sh` from the venv again.

**Reply by dorkmo** [reply]
thanks! i had a feeling something was amiss, wasnt showing the new Network Name line in the full node status.

**Reply by Bronkow** [reply]
Thanks! Everything works well now

**Comment by dorkmo** [comment]
seems my pi gui node is running between 3 to 7 minutes behind. was running okay after flag, but at around 2:45 pacific time attempted proofs hiccuped . It made two more attempted proofs, with the last at 3:53, nothing since. Standing by to switch to plan B and farm on windows.

Anyone having luck on the pi gui?

**Reply by dorkmo** [reply]
I lowered the number of max peers in config.yaml hoping that might reduce the load on the pi? Restarted and running again. Will see if it can hang in there.

**Reply by Bronkow** [reply]
Hej dorkmo, could you Fix the Time Problem? My pi has the same issue.

**Reply by dorkmo** [reply]
seems to be working so far. not sure if it was that or something else. You might give it a try to see how you do?!

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
